### PR TITLE
Fix #8 - Replace string interpolation that are not f strings

### LINKED ISSUE #8: use only f strings for string interpolation
The method of string interpolation should be standardised to [f strings](https://realpython.com/python-f-strings/), as this is a Python 3 repo.

There was a point at which some of the scripts were confused between Python 2 and 3, which is partly why we have a range of interpolation methods.

### PR DESCRIPTION
**Pull request by cuichenli**
closes #8

### CODE REVIEW

**Comment by breezykermo** [comment]
Thanks so much for this! My apologies, I neglected to mention in the issue that all interpolations should be replaced with f strings **except for those in the top level run.py**.

This is because run.py is the only file that is run using the local Python installation (the rest are run inside the docker container that run.py spins up), and so it should accommodate both Python 2 and Python 3 to be more flexible for the time being.

Would you be able to revert the interpolations in run.py to the `.format()` syntax? Then I am happy to merge. Thanks again!

**Comment by cuichenli** [comment]
@breezykermo No worries. Just reverted.

File: src/lib/selectors/youtube/select.py [hunk 1]
```diff
@@ -83,15 +83,14 @@ def str2bool(v):
 def youtube_search_all_pages(args, logs):
     csv_obj = []
     print_log(
-        "Search terms: %s\n Start: %s\n End: %s"
-        % (args["q"], args["after"], args["before"]),
+        f"Search terms: {args["q"]}\n Start: {args["after"]}\n End: {args["before"]}",
```
Comment (breezykermo) [inline src/lib/selectors/youtube/select.py]: I've tried running this, and it introduces errors due to the nested double quotes across this file. (This is partly our fault, as we don't yet have tests and CI setup-- i.e. #12, #13, #14, plus the configuration documentation needs some work.) Would you be able to change this and the other instances in this file?

For example, this should become:
```python
f"Search terms: {args['q']}\n Start: {args['after']}\n End: {args['before']}",
```

**Review by breezykermo — CHANGES_REQUESTED** [review]

File: src/lib/selectors/youtube/select.py (hunk 1 above)
Comment (cuichenli) [reply src/lib/selectors/youtube/select.py]: You are absolutely right, this should be changed.

**Comment by breezykermo** [comment]
Sorry about the hassle here, I've opened a priority issue to add CI and better developer onboarding, will hopefully be a much smoother experience by tomorrow or the next day.

**Comment by cuichenli** [comment]
It is totally ok, actually if you want I can also help to set up the ci pipelines.

**Comment by breezykermo** [comment]
Sure thing, thanks! Take a look at #12 and #16, both of which you may be able to weigh in on in particular if you've already struggled through setting this repo up to run.

File: src/lib/selectors/youtube/select.py (hunk 1 above)
Comment (breezykermo) [reply src/lib/selectors/youtube/select.py]: Note that there are a couple of other instances in this file, youtube/select.py, where this same nested double quote needs to be fixed. Sorry if my comment before wasn't clear!

**Comment by cuichenli** [comment]
Actually, have not run it yet (noticed I may need some API credentials to make it). So I barely went through the procedures. Gonna first manage to set up the pipeline.

File: src/lib/selectors/youtube/select.py (hunk 1 above)
Comment (cuichenli) [reply src/lib/selectors/youtube/select.py]: My bad. Should have checked it at first.

Comment (cuichenli) [reply src/lib/selectors/youtube/select.py]: Applied flake8 and pylint this time. Hope it will be all good. 😅

**Comment by breezykermo** [comment]
@cuichenli would you be able to rebase onto master? We've just merged a PR that should allow you to get up and running to test your changes. CI and a test suite coming soon.

**Review by breezykermo — APPROVED** [review]

**Pull request merged**